packages/jest-config/src/resolveConfigPath.ts

```diff
@@ -95,3 +108,12 @@ const makeResolutionErrorMessage = (
   `traverse directory tree up, until it finds one of those files in exact order: ${JEST_CONFIG_EXT_ORDER.map(
     ext => `"${getConfigFilename(ext)}"`,
   ).join(' or ')}.`;
+
+const multipleConfigsWarn =
+  '● Multiple configurations found:\n\n' +
+  'Jest will use `path/to/jest.config.js` for configuration, but Jest also\n' +
+  'found configuration in `path/to/package.json`. Delete the `"jest"` key\n' +
```

Comment:
these should point to the actual files instead of a generic `path/to/jest.config.js`